piccolo_api/media/local.py

```diff
@@ -95,7 +95,7 @@
             with open(path, "wb") as new_file:
                 shutil.copyfileobj(file, new_file)
                 if file_permissions is not None:
-                    os.chmod(path, 0o640)
+                    os.chmod(path, 0o644)
```

Comment:
@dantownsend This is the only warning left so I think this might work. I think the previous permission is also ok but I change your permission to `644` based on [this](https://www.namecheap.com/support/knowledgebase/article.aspx/400/205/file-permissions/), but that is not enough either. Any idea what file permission would satisfy `codeql` so that the Piccolo API code would still work. Thanks.